portal/src/app/about/AboutContent.jsx

```diff
@@ -11,19 +12,99 @@ class AboutContent extends React.Component {
+                <p className="AboutContent_paragraph">
+                    SEARCH GENOME tab: The iMADS database contains genome-wide predictions for human genome versions
+                    hg19 and hg38, for eleven TFs and ten TF pairs. These predictions can be visualized in the UCSC
+                    Genome Browser, using the track hubs <a href="http://trackhub.genome.duke.edu/gordanlab/tf-dna-binding-predictions/hub.txt">
+                        trackhub.genome.duke.edu/gordanlab/tf-dna-binding-predictions/hub.txt
+                    </a> and <a href="http://trackhub.genome.duke.edu/gordanlab/tf-dna-preferences/hub.txt">
+                        trackhub.genome.duke.edu/gordanlab/tf-dna-preferences/hub.txt
+                    </a>, or directly in the iMADS
+                    web application, using the graphical interface. Users can explore predictions around transcription
+                    start sites (TSSs) of gene, using preloaded or custom gene lists, as well as custom lists of
+                    genomic coordinates.
+                </p>
             </div>
-            <br />
             <div>
-                <span className="AboutContent_source_code_label">Source Code</span>
+                <p className="AboutContent_paragraph">
+                MAKE PREDICTIONS tab: Users can also provide custom sequences as input to obtain predictions according
+                    to any binding specificity or differential binding specificity (preference) model in our framework.
+                </p>
+            </div>
+            <p className="AboutContent_paragraph">
+                All predictions can be downloaded in tab-delimited and csv formats. All data sources used in the iMADS
+                web application are described in the DATA SOURCES tab.
+            </p>
+            <div>
+                <span className="AboutContent_group_label">TECHNOLOGY</span>
+                <ul>
+                    <li>Database management system: PostgreSQL</li>
+                    <li>Programming languages: python, R, JavaScript</li>
+                    <li>Web framework: Flask</li>
+                    <li>Technical documentation also available at <a href="https://github.com/Duke-GCB/iMADS/blob/master/README.md">
+                        github.com/Duke-GCB/iMADS/blob/master/README.md
+                        </a>
+                    </li>
+                </ul>
+            </div>
+            <div>
+                <span className="AboutContent_group_label">SOURCE CODE</span>
                 <ul>
-                    <li>Website: {websiteSource}</li>
-                    <li>TF binding predictions: {predictionSource}</li>
-                    <li>TF preference: {preferenceSource}</li>
+                    <li>Website: <a href="https://github.com/Duke-GCB/iMADS">github.com/Duke-GCB/iMADS</a></li>
+                    <li>iMADS Worker: <a href="https://github.com/Duke-GCB/iMADS-worker">
```

Comment:
What the _iMADS Worker_ does hasn't been mentioned yet. Use a more descriptive label, such as _Worker for prediction server_?